Changelog

```diff
@@ -5,19 +5,308 @@
 ================
 
 This document contains change notes for bugfix releases in
-the 4.0.x series (latentcall), please see :ref:`whatsnew-4.0` for
-an overview of what's new in Celery 4.0.
+the 4.1.x series (latentcall), please see :ref:`whatsnew-4.1` for
```

Comment:
Well comparing to previous releases it seems like it :smile: :
- Cipater
- Chiastic Slide